packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,646 @@
+              "type": "string"
+            },
+            "x-taplo": {
+              "docs": {
+                "main": "Additional request headers to send in [proxy redirects](https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects)."
+              },
+              "links": {
+                "key": "https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects"
+              }
+            }
+          },
+          "signed": {
+            "title": "Signed",
+            "description": "Name of an environment variable for signed proxy redirects.",
+            "type": "string",
+            "x-taplo": {
+              "docs": {
+                "main": "Name of an environment variable for [signed proxy redirects](https://docs.netlify.com/routing/redirects/rewrites-proxies/#signed-proxy-redirects)."
+              },
+              "links": {
+                "key": "https://docs.netlify.com/routing/redirects/rewrites-proxies/#signed-proxy-redirects"
+              }
+            }
+          }
+        },
+        "additionalProperties": false,
+        "required": ["from"],
+        "x-taplo": {
+          "links": {
+            "key": "https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file"
+          }
+        }
+      }
+    },
+    "headers": {
+      "type": "array",
+      "items": {
+        "title": "Headers",
+        "description": "Define custom headers for specific paths.",
+        "type": "object",
+        "properties": {
+          "for": {
+            "title": "For",
+            "description": "Define which paths this specific headers block will cover.",
+            "type": "string"
+          },
+          "values": {
+            "title": "Values",
+            "description": "Define the actual headers.",
+            "type": "object",
+            "additionalProperties": {
+              "type": "string"
+            }
+          }
+        },
+        "required": ["for"],
+        "additionalProperties": false
+      },
+      "x-taplo": {
+        "docs": {
+          "main": "Additional request headers to send in [proxy redirects](https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects)."
+        },
+        "links": {
+          "key": "https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects"
+        }
+      }
+    },
+    "functions": {
+      "title": "Functions",
+      "description": "Although there are default settings for Netlify Functions to help you get started, you can use this section for optional, custom configuration.",
+      "type": "object",
+      "additionalProperties": {
+        "description": "Options that only apply for this filter by name, including using glob patterns. If a function matches several configuration blocks containing one of these properties, the values are concatenated.",
```

I'm not sure I understand this one but I think I found what your referencing. Here's a suggested edit to make it clearer:

```suggestion
        "description": "Set these options for all functions or filter them by function name, including using glob patterns. If a function matches several configuration blocks containing one of these properties, the values are concatenated.",
```

If this is only for specific function names and not all:

```suggestion
        "description": "Set these options for specific functions by filtering them by function name. You can use glob patterns in your filter. If a function matches several configuration blocks containing one of these properties, the values are concatenated.",
```